src/index.html

```diff
@@ -0,0 +1,1302 @@
+              <figcaption class="flex">
+                <a href="#">
+                  KeepSolid VPN Unlimited: Lifetime Subscription
+                </a>
+                <span class="flex"> <span class="line-through">$199</span> $39 </span>
+              </figcaption>
+            </figure>
+          </article>
+          <article class="flex-box flex-deals">
+            <figure>
+              <div class="article-img-deals flex">
+                <img src="../src/assets/rocket.png" alt="picture of a rocket launching.">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  LongTailPro: One Time (10,000 Keyword Lookups. No Expiry, No Reset)
+                </a>
+                <span class="flex"> <span class="line-through">$1500</span> $49.99 </span>
+              </figcaption>
+            </figure>
+          </article>
+          <article class="flex-box flex-deals">
+            <figure>
+              <div class="article-img-deals flex">
+                <img src="../src/assets/StackSkills.png" alt="picture of an website featuring a tutorial">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  StackSkills Unlimited: Lifetime Access
+                </a>
+                <span class="flex"> <span class="line-through">$1495</span> $59 </span>
+              </figcaption>
+            </figure>
+          </article>
+          <article class="flex-box flex-deals">
+            <figure>
+              <div class="article-img-deals flex">
+                <img src="../src/assets/VPN-1.png" alt="picture of VPN by VeePN">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  VPN by VeePN: Lifetime Subscription
+                </a>
+                <span class="flex"> <span class="line-through">$659</span> $29.99 </span>
+              </figcaption>
+            </figure>
+          </article>
+          <article class="flex-box flex-deals">
+            <figure>
+              <div class="article-img-deals flex">
+                <img src="../src/assets/VPN-3.png" alt="picture of  Ivacy VPN">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  Ivacy VPN: Lifetime Subscription
+                </a>
+                <span class="flex"> <span class="line-through">$1194</span> $39.99 </span>
+              </figcaption>
+            </figure>
+          </article>
+        </div>
+      </section>
+    </main>
+
+    <footer>
+      <div class="top-footer">
+        <div class="social flex">
+          <ul class="flex">
+            <li>
+              <a href="#">
+                <i class="fab fa-facebook-square"></i>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <i class="fab fa-twitter"></i>
+              </a>
+            </li>
```

Comment:
- [ ] Address the whitespace in the bottom (should not exist).

![image](https://user-images.githubusercontent.com/61479648/89059358-6aff4e00-d337-11ea-9fa1-c5b378b3d154.png)